File: cypress-homework/cypress/support/payloads.js
const {faker} = require('@faker-js/faker');

function validGoalPayload() {
    return {
        multiple_owners: true,
        name: faker.company.catchPhrase(),
        due_date: Date.now() + 7 * 24 * 60 * 60 * 1000,
        description: faker.lorem.sentence(),
        color: faker.color.rgb({prefix: '#'})
    };
}

function invalidTeamId() {
    return Cypress._.random(
        100000000000000,
        999999999999999
    ).toString();
}

module.exports = {
    validGoalPayload,
    invalidTeamId
};

File: python-homework/utils/payloads.py
from faker import Faker
import time

faker = Faker()


def valid_goal_payload():
    return {
        "multiple_owners": True,
        "name": faker.company(),
        "due_date": int(time.time() * 1000) + (7 * 24 * 60 * 60 * 1000),
        "description": faker.sentence(),
        "color": faker.hex_color()
    }